webCommander/centralHub/views.py

```diff
@@ -1,7 +1,29 @@
 from django.shortcuts import render,get_object_or_404
-from centralHub.models import TwitterUser
+from .forms import SubmitXUser
+from centralHub.models import TwitterUser, TwitterUserPosts
```

Comment:
this can also be a relative import, nicer to mirror the other two

Reply:
done

Reply:
yep or all absolute paths, that's cool too. 
relative imports are a bit more flex if you'd rename the app, but that's unlikely to happen anyways.